lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -1804,14 +1793,6 @@ bool OCLBackend::isOpSupported(const NodeInfo &NI) const {
                                                   {ConvolutionNode::BiasIdx}) &&
            (NI.getInElemTy(ConvolutionNode::BiasIdx) == ElemKind::Int32QTy);
 
-  case Kinded::Kind::OCLConvolutionNodeKind:
```

Comment:
for convolution: we do not support NCHW with group > 1.

Reply:
True, but the node doesn't get Transformed if group is > 1. That's also a runtime param that can't be checked here in isOpSupported.